chaosaws/ec2/actions.py

```diff
@@ -21,10 +22,10 @@ def stop_instance(instance_id: str = None, az: str = None, force: bool = False,
     """
     Stop a single EC2 instance.
 
-    You may provide an instance id explicitely or, if you only specify the AZ,
+    You may provide an instance id explicitly or, if you only specify the AZ,
     a random instance will be selected. If you need more control, you can
     also provide a list of filters following the documentation
-    https://boto3.readthedocs.io/en/latest/reference/services/ec2.html#EC2.Client.describe_instances
+    https://bit.ly/2Sv9lmU
```

Comment:
Not sure there was a need for these shorter links ;)

Even if the line is now shorter I liked knowing the source of truth.